partiql-conformance-tests/Cargo.toml

```diff
@@ -28,9 +28,10 @@ required-features = ["report_tool"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [build-dependencies]
-walkdir = "2.3"
-ion-rs = "0.6.0"
-codegen = "0.1.3"
+#walkdir = "2.3"
+#ion-rs = "0.6.0"
+#codegen = "0.1.3"
```

Comment:
Think these dependencies can be deleted

Reply:
yep; removed already in the next set of commits for the next PR